File: problems/pset_1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

// Defining Functions
int get_number(string prompt);

int main(void)
{
    // Saving Height as int var
    int height = get_number("Height: ");
    for (int i = 0; height > i; i++)
    {

        // Get Number of Spaces
        for (int spaces = height - i - 2; spaces >= 0; spaces--)
        {
            printf(" ");
        }

        // Get Number of Hashes
        for (int hashes = 0; hashes <= i; hashes++)
        {
            printf("#");
        }

        // Starting new Line
        printf("\n");

    }
}

// Get Height
int get_number(string prompt)
{
    int height;
    do
    {
        height = get_int("%s", prompt);
    }
    while (height < 1 || height > 8);
    return height;
}

File: problems/pset_6/mario/less/mario.py
from cs50 import get_int


def main():
    # Get Height
    height = get_int("Height: ")
    if height > 0 and height < 9:
        for i in range(height):

            # Get Number of Hashes
            hash = i + 1
            hash_string = "#" * hash

            # Get Number of Spaces
            space = abs(i + 1 - height)
            space_string = " " * space

            # Print
            print(space_string+hash_string)

    else:
        # Run main Again if height is incorrect
        main()


main()